config.clj

```diff
@@ -96,9 +96,14 @@
   [color/white color/red color/white color/yellow color/white color/blue])
 (def american-colors
   [color/blue color/blue color/blue color/white color/red color/white color/red color/white color/red color/white])
+(def rainbow-colors
+  (map color/rainbow (range 0 1 0.1)))
+(def fire-colors
+  (let [colors [(color/rgb 156/256 42/256 34/256) (color/rgb 226/256 87/256 34/256) (color/rgb 1.0 73/256 73/256) (color/rgb 1.0 189/256 91/256) (color/rgb 253/256 207/256 88/256)]]
+    (repeatedly 12 #(rand-nth colors))))
```

Comment:
OK, fair enough on the style nit. This was a refactoring of a refactoring of an initial attempt that was *much* more stylistically objectionable ;) So would something like this be more idiomatic?

```clojure
(def fire-colors
  (repeatedly 12 #(rand-nth
                    [(color/rgb 156/256 42/256 34/256)
                     (color/rgb 226/256 87/256 34/256)
                     (color/rgb 1.0 73/256 73/256)
                     (color/rgb 1.0 189/256 91/256)
                     (color/rgb 253/256 207/256 88/256)]))))
```

And yes, the randomization on each boot was my (ridiculously simplistic) way to add "entropy" to this mode. This was a really spur-of-the-moment thought I had as I was wrapping up last night: what is the most ridiculously simple way I can try to make a vaguely fire-like effect with one of the existing modes, in 10 minutes' coding or less? If you'd rather just make it totally static, or remove it entirely, that's cool. I also have no sense of how well/badly this effect will work on the actual lantern, so might be worth looking at that and then making a call.